### DISCUSSION TITLE
Feather Huzzah esp8266 compatibility

### DISCUSSION THREAD

**Discussion by Mitko939** [arendst/Tasmota#14605] [Q&A] [answered]
Hi, I have installed tasmota ir on my huzzah. The connection to the card via tasmota works, but I can't get any pins to work. For example I set all the pins as ir receiver and connected an ir receiver, but i can't get any ir signal in the console (i also tried with wemos d1 mini and everything works). is there any way to make esp8266 huzzah feather work? I have not found anything on the net. is there any template available just for this card?

**Comment by Jason2866** [comment, marked as answer]
Looking at the schematic diagramm of the [huzzah](https://cdn-learn.adafruit.com/assets/assets/000/088/838/original/adafruit_products_Feather_ESP8266_HUZZAH_sch.png?1583183441) i can see nothing why it should not work with a standard Tasmota build
Can you post a photo of the board, maybe it is a different board?

**Reply by sfromis** [reply]
If you "set all the pins as ir receiver and connected an ir receiver", it won't work, it has to be *one at a time*.
